schema/mqtt-subscribe.json

```
@@ -0,0 +1,28 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "type": "object",
+  "required": ["machineId, siteId, timestamp"],
+  "properties": {
+    "machineId": {
+      "type": "string",
+      "format": "uuid"
+    },
+    "siteId": {
+      "type": "string",
+      "format": "uuid"
+    },
+    "latitude": {
+      "type": "number",
+      "minimum": -90,
+      "maximum": 90
+    },
+    "longitude": {
+      "type": "number",
+      "minimum": -180,
+      "maximum": 180
+    },
+    "timestamp": {
+      "type": "number"
```

Lisäisin 
```suggestion
      "type": "number",
      "description": "Timestamp in milliseconds since epoch"
```
Olihan tämä ms?

---

lisäksi, näköjään toisaalla apissa on käytetty termiä timestampMs. Pitäisi yhtenäisyyden vuoksi käyttää samaa tässä?

Pahoittelut, enpä hoksannut tätä katsoa kun rustasin ehdotusta formaatista